### PR TITLE
Proofreading task - File about Search

### PR DESCRIPTION
**Pull request by johannahemminger**
Some changes to the Data Hub Catalog Search document.

### CODE REVIEW

File: content/data-hub/data-hub-catalog/search.md [hunk 1]
```diff
@@ -410,7 +410,7 @@ Technical owners can be [changed](curate#changing-owners).
 
 When a data source is registered, by default, it is **Discoverable** in the Data Hub Catalog. When this is set, all users in your company can find it, view the details, and consume it. The owners of an asset and curators can set a data source as **Non-discoverable**, which means it is not visible to users unless they are the owner or a curator.
 
-See [Curate Bar](#curate-bar) for changing **Discoverability** as the owner of the data source or curator.
+See [Curation Bar](#curation-bar) for changing **Discoverability** as the owner of the data source or curator.
```
Comment (Adam-Dupaski) [inline content/data-hub/data-hub-catalog/search.md:413]: Please review SG for best practices on wording a cross-reference sentence.

File: content/data-hub/data-hub-catalog/search.md [hunk 2]
```diff
@@ -433,13 +433,9 @@ The environment type indicates the quality and the status of the data that the e
...
 
-### 3.7 Curate Bar {#curate-bar}
+### 3.7 Curation Bar {#curation-bar}
 
-The **Curate Bar** is displayed in the asset detail screen if you are the owner of the selected asset or a curator:
-
-![called out curate bar](attachments/search/curate-bar.png)
-
-In **Edit** > **Edit Metadata**, you can edit the information that is displayed in the Catalog for an asset:
+The **Curation Bar** is displayed in the asset detail screen if you are the owner of the selected asset or a curator. In **Edit** > **Edit Metadata**, you can edit the information that is displayed in the Catalog for an asset:
```
Comment (Adam-Dupaski) [inline content/data-hub/data-hub-catalog/search.md:438]: If **Curation Bar** is the correct term, are there changes in other docs necessary?

File: content/data-hub/data-hub-catalog/search.md [hunk 3]
```diff
@@ -448,20 +444,20 @@ For further details, see [Curate Registered Assets](curate).
...
+The data source URI is the location of the service contract of the data source, also known as the service endpoint. The endpoints of all exposed datasets (entity sets) are defined in the contract. From the details screen of the data source and dataset, you can copy the URIs to the clipboard by clicking the **Copy Data Source URI** and **Copy Dataset URI** respectively. These URIs can be used for directly accessing the contract and resource in BI applications.
 
 ### 3.9 Download the Metadata Contract of a Data Source {#download-contract}
 
-For a selected data source, you can click **Download** to download the OData service contract that is located at the data source endpoint. A ZIP file that includes the all the files that make up the full metadata contract is generated and downloaded.
+For a selected data source, you can click **Download** to download the OData service contract that is located at the data source endpoint. A *.zip* file that includes the all the files that make up the full metadata contract is generated and downloaded.
```
Comment (Adam-Dupaski) [inline content/data-hub/data-hub-catalog/search.md:451]: Please review SG for entry on "ZIP" specifically.

Comment (johannahemminger) [reply content/data-hub/data-hub-catalog/search.md:451]: Ah OK, didn't see this in there at first. Thanks

**Review by Adam-Dupaski — APPROVED** [review]

**Pull request merged**